src/internet/hello_web.cpp

```diff
@@ -168,7 +168,8 @@ namespace License
 		curl_easy_setopt(x, CURLOPT_VERBOSE, 1); 
 		curl_easy_setopt(x, CURLOPT_STDERR, pFile); 
 		auto code = curl_easy_perform(x);
-		fclose(pFile);
+		//TODO : removed because cause crashes on windows (maybe related to curl version tested (7.53.1) : does curl close file handle ?
+		//fclose(pFile);
```

Comment:
strange? Do we risk a a file handle leak?

Reply:
I don't know why it crashes on windows only.
I propose to create an issue with this TODO.